### ISSUE TITLE
Jellyfin server, some more clients

### ISSUE THREAD

**Issue by Extarys** [basings/selfhosted-music-overview#79]
Emby is in the server part, but Jellyfin is missing :cry: 

Gelli for Jellyfin was added as a client tho  https://github.com/basings/selfhosted-music-overview/pull/45

There are also some other clients missing.

- [ ] Server: Jellyfin
- [ ] Client: Tauon ([Desktop](https://tauonmusicbox.rocks/))
- [ ] Client: S2 ([Android](**url**)) for Jellyfin and Emby

**Comment by m0ngr31** [comment]
Jellyamp is also another client

**Comment by basings** [comment]
Thx for the heads up of jellyfin! fixed in #81

**Comment by basings** [comment]
I added #82 

I actually want to remove all servers/clients from the table that have no info and leave it as an issue until the info is filled

**Comment by basings** [comment]
@Extarys do you mean [https://blog.shuttlemusicplayer.app/](https://blog.shuttlemusicplayer.app/)?

**Comment by basings** [comment]
#84

**Comment by Extarys** [comment]
@basings Yes, S2 player homepage seems to be https://blog.shuttlemusicplayer.app/ indeed. If I research the information can I leave it here and someone make a PR? The Markdown table editing gives me bad headaches :P

**Comment by basings** [comment]
@Extarys sure!

editing markdown tables is pretty easy!

with Typora you can edit markdown WYSIWYG :)

**Comment by basings** [comment]
I'll add a template to issue #83 if you want to populate it yourself, but if it's too much asked, I'm happy to fill out the table as well!

**Comment by basings** [comment]
added template to #83

**Comment by Extarys** [comment]
thanks! I ended up copying the template from the README and use Kate without word wrap to fill it for [Tauon](https://github.com/basings/selfhosted-music-overview/issues/82). I'll try Typora for the next one.

Let me know if I'm doing this wrong.

**Comment by basings** [comment]
that was perfect!! thx =)
